src/components/playbackManager.ts

```diff
@@ -204,46 +223,92 @@ export class playbackManager {
         loadRequestData.media = mediaInfo;
         loadRequestData.autoplay = true;
 
+        // If we should seek at the start, translate it
+        // to seconds and give it to loadRequestData :)
+        if (mediaInfo.customData.startPositionTicks > 0)
+            loadRequestData.currentTime =
+                mediaInfo.customData.startPositionTicks / 10000000;
```

Comment:
Don't we already have a function for translating ticks to seconds?

Reply:
No, we don't actually. I see many places where this division/multiplication is going on. Good idea for a helper function though.

Reply:
Helper created [here](https://github.com/hawken93/jellyfin-chromecast/pull/228/commits/ffebe175a3f9fbca7180504af9e2d3828a7f5b48) ... is that an appropriate place for the constant or does it belong somewhere else?